action.yml

```diff
@@ -1,13 +1,13 @@
-name: 'Notify for pending release'
+name: 'Github Action Notify Release'
 description: 'Create an issue if there is a release pending'
 inputs:
   github-token:
     description: 'GitHub token'
     required: true
-  days-to-stale-release:
-    description: 'The number of days after which notification should trigger after last unreleased commit'
-    required: true
+  stale-days:
+    description: 'Number of days of inactivity before a release becomes stale'
+    required: false
     default: 7
 runs:
-  using: 'node12'
+  using: 'node14'
```

Comment:
this is the problem with node14. Here it is not supported, in CI it is supported.